assets/authchoice.js

```diff
@@ -28,7 +28,7 @@ jQuery(function($) {
         return this.each(function() {
             var $container = $(this);
 
-            $container.find('a').on('click', function(e) {
+            $container.find('a.auth-link').on('click', function(e) {
```

Comment:
This would not work in case someone creates link with custom CSS class as `auth-link` added only in case no other class is specified.